cpp/src/arrow/util/compression_test.cc

```diff
@@ -375,10 +375,11 @@ TEST(TestCodecMisc, SpecifyCompressionLevel) {
     bool expect_success;
   };
   constexpr CombinationOption combinations[] = {
-      {Compression::GZIP, 2, true},     {Compression::BROTLI, 10, true},
-      {Compression::ZSTD, 4, true},     {Compression::LZ4, -10, false},
-      {Compression::LZO, -22, false},   {Compression::UNCOMPRESSED, 10, false},
-      {Compression::SNAPPY, 16, false}, {Compression::GZIP, -992, false}};
+      {Compression::GZIP, 2, true},       {Compression::BROTLI, 10, true},
+      {Compression::ZSTD, 4, true},       {Compression::LZ4, -10, false},
+      {Compression::LZO, -22, false},     {Compression::UNCOMPRESSED, 10, false},
+      {Compression::SNAPPY, 16, false},   {Compression::GZIP, -992, false},
+      {Compression::LZ4_FRAMED, 9, true}, {Compression::LZ4_FRAMED, -99, false}};
```

Comment:
```suggestion
      {Compression::LZ4_FRAME, 9, true}, {Compression::LZ4_FRAME, -99, false}};
```

Reply:
Fixed, and also removed the testcase with invalid value, because LZ4 will rectify that internally (and not fail)